### PR TITLE
Update values.py added both 2018 Camry LE and RAV4 XLE

### PR DESCRIPTION
**Pull request by monterman**
Update values.py  by adding both 2018 Camry LE and RAV4 XLE, the fingerprints in mine are different than the ones in 0.5.6.

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -116,11 +120,15 @@ def check_ecu_msgs(fingerprint, ecu):
...
+  },
   #XLE and LE
   {
     36: 8, 37: 8, 119: 6, 170: 8, 180: 8, 186: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 550: 8, 552: 4, 562: 6, 608: 8, 610: 8, 643: 7, 658: 8, 705: 8, 728: 8, 740: 5, 761: 8, 764: 8, 800: 8, 810: 2, 812: 8, 814: 8, 818: 8, 822: 8, 824: 8, 830: 7, 835: 8, 836: 8, 869: 7, 870: 7, 871: 2, 888: 8, 889: 8, 891: 8, 898: 8, 900: 6, 902: 6, 905: 8, 918: 8, 921: 8, 933: 8, 934: 8, 935: 8, 944: 8, 945: 8, 951: 8, 955: 8, 956: 8, 976: 1, 983: 8, 984: 8, 998: 5, 999: 7, 1000: 8, 1001: 8, 1002: 8, 1011: 8, 1014: 8, 1017: 8, 1020: 8, 1041: 8, 1042: 8, 1044: 8, 1056: 8, 1059: 1, 1076: 8, 1077: 8, 1082: 8, 1114: 8, 1161: 8, 1162: 8, 1163: 8, 1164: 8, 1165: 8, 1166: 8, 1167: 8, 1228: 8, 1235: 8, 1237: 8, 1263: 8, 1264: 8, 1279: 8, 1412: 8, 1541: 8, 1552: 8, 1553: 8, 1556: 8, 1557: 8, 1568: 8, 1570: 8, 1571: 8, 1572: 8, 1595: 8, 1745: 8, 1779: 8, 1786: 8, 1787: 8, 1788: 8, 1789: 8, 1808: 8, 1816: 8, 1904: 8, 1912: 8, 1990: 8, 1998: 8
   },
-  #XSE and SE
+  #XLE and LE (NOT MINE!)
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: why this change?

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -116,11 +120,15 @@ def check_ecu_msgs(fingerprint, ecu):
...
   }],
   CAR.CAMRY: [
+  #CAR.CAMRYLE_2018: [{ (MINE)
+  {
+    36: 8, 37: 8, 119: 6, 170: 8, 180: 8, 186: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 550: 8, 552: 4, 562: 6, 608: 8, 610: 8, 643: 7, 705: 8, 728: 8, 740: 5, 761: 8, 764: 8, 800: 8, 824: 8, 830: 7, 835: 8, 836: 8, 869: 7, 870: 7, 898: 8, 905: 8, 921: 8, 933: 8, 944: 8, 945: 8, 951: 8, 955: 8, 956: 8, 976: 1, 1002: 8, 1017: 8, 1020: 8, 1041: 8, 1042: 8, 1056: 8, 1059: 1, 1076: 8, 1077: 8, 1114: 8, 1164: 8, 1165: 8, 1166: 8, 1167: 8, 1279: 8, 1541: 8, 1552: 8, 1553: 8, 1556: 8, 1568: 8, 1570: 8, 1571: 8, 1572: 8, 1595: 8, 1745: 8, 1779: 8, 1904: 8
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: this is already fully included in the next one (line 129), so it's not needed

File: selfdrive/car/toyota/values.py [hunk 3]
```diff
@@ -84,7 +84,11 @@ def check_ecu_msgs(fingerprint, ecu):
   CAR.RAV4H: [{
     36: 8, 37: 8, 170: 8, 180: 8, 186: 4, 296: 8, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 547: 8, 548: 8, 550: 8, 552: 4, 560: 7, 562: 4, 581: 5, 608: 8, 610: 5, 643: 7, 705: 8, 713: 8, 725: 2, 740: 5, 800: 8, 835: 8, 836: 8, 849: 4, 869: 7, 870: 7, 871: 2, 896: 8, 897: 8, 900: 6, 902: 6, 905: 8, 911: 8, 916: 3, 918: 7, 921: 8, 933: 8, 944: 8, 945: 8, 950: 8, 951: 8, 953: 3, 955: 8, 956: 8, 979: 2, 998: 5, 999: 7, 1000: 8, 1001: 8, 1005: 2, 1008: 2, 1014: 8, 1017: 8, 1041: 8, 1042: 8, 1043: 8, 1044: 8, 1056: 8, 1059: 1, 1114: 8, 1161: 8, 1162: 8, 1163: 8, 1176: 8, 1177: 8, 1178: 8, 1179: 8, 1180: 8, 1181: 8, 1184: 8, 1185: 8, 1186: 8, 1190: 8, 1191: 8, 1192: 8, 1196: 8, 1197: 8, 1198: 8, 1199: 8, 1212: 8, 1227: 8, 1228: 8, 1232: 8, 1235: 8, 1237: 8, 1263: 8, 1264: 8, 1279: 8, 1408: 8, 1409: 8, 1410: 8, 1552: 8, 1553: 8, 1554: 8, 1555: 8, 1556: 8, 1557: 8, 1561: 8, 1562: 8, 1568: 8, 1569: 8, 1570: 8, 1571: 8, 1572: 8, 1584: 8, 1589: 8, 1592: 8, 1593: 8, 1595: 8, 1596: 8, 1597: 8, 1600: 8, 1656: 8, 1664: 8, 1728: 8, 1745: 8, 1779: 8, 1904: 8, 1912: 8, 1990: 8, 1998: 8
   },
-  # Chinese RAV4
+  # CAR.RAV4 XLE_2018: (MINE!)
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: is your car hybrid? doe snot seem like, as this is a subset of `CAR.RAV4`

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (monterman) [reply selfdrive/car/toyota/values.py]: those were my comments. lol.  my bad.
best to remove...

File: selfdrive/car/toyota/values.py (hunk 3 above)
Comment (monterman) [reply selfdrive/car/toyota/values.py]: yes it is a subset of Rav4

File: selfdrive/car/toyota/values.py (hunk 2 above)
Comment (monterman) [reply selfdrive/car/toyota/values.py]: note, this is a different fingerprint for the CAMRY LE.  I did a compare with the other fingerprint on line 128 and the are different.

Do add this fingerprint as a subset for Camry LE 2018.

File: selfdrive/car/toyota/values.py (hunk 3 above)
Comment (monterman) [reply selfdrive/car/toyota/values.py]: this fingerprint is also new and was not in the values.py before.

My apologies for the error in placing in the wring subset,  I did not realize it.

**Comment by rbiasini** [comment]
both added fingerprints are strictly included in existing ones.
